docassemble/AssemblyLine/al_document.py

```diff
@@ -36,25 +42,20 @@ def html_safe_str(the_string) -> str:
   """
   return re.sub( r'[^A-Za-z0-9]+', '_', the_string )
 
-def table_row( aldoc, key='final', view_icon="eye", download_icon="download", format="pdf", refresh=True ):
+def table_row( title, view_file:DAFile, download_file:DAFile=None, view_icon:str="eye", download_icon:str="download") -> str:
   """
   Return a string of html that is one row of a table containing
   the `.as_pdf()` contents of an AL object and its interaction buttons
```

Comment:
This comment is no longer up to date, since you're directly passing in view and download files.